apps/keyboard/js/views/base_view.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+(function(exports) {
+
+/**
+ * BaseView is the base of all simple views in the keyboard app.
+ * It provides the common interface like, highlight() and hide().
+ */
+function BaseView(target, options, viewManager) {
+  this.target = target;
+  this.options = options;
+  this.viewManager = viewManager;
+}
+
+exports.BaseView = BaseView;
```

Comment:
Let's add a `BaseView.prototype.element = null;` here and make a comment on this property.